style.css

```diff
@@ -331,3 +331,79 @@ ul {
   font-size: 22px;
   line-height: 43px;
 }
+
+/* ---- contact form ---- */
+
+.contact-form-section {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  min-height: 100vh;
+  align-items: center;
+  gap: 50px;
+  padding-top: 120px;
+}
+
+.contact-form-section h6,
+form {
+  width: 80%;
+}
+
+.contact-form-section form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+
+.contact-form-section h6 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-form-section label {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
+
+#fs-frm-inputs {
+  display: flex;
+  flex-direction: column;
+  border: none;
+  gap: 10px;
+  width: 100%;
+}
+
+#fs-frm-inputs input,
+textarea {
+  border-style: none none solid none;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 0 0 10px 10px;
+}
```

Comment:
- [ ] Kindly make sure to give all `input` and `textarea` fields more space in order to look like the provided Figma design. :wink: 

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/199319472-a52587e6-eb8f-4d83-b9cc-b6358e3c7df7.png)

   - ### Figma design
  ![image](https://user-images.githubusercontent.com/87197412/199319549-e00612f7-5a73-48c5-ba2b-87cc8ba8966a.png)